### ISSUE TITLE
Worth discussing non-Gregorian calendar support in Moment alternatives

### ISSUE THREAD

**Issue by bcullman** [you-dont-need/You-Dont-Need-Momentjs#126]
non-Gregorian calendars (Hebrew, Islamic, Solar, Japanese and a few others) are one of the reasons in the past my teams had to stick with Moment. 

It would be interesting to have a table or column in existing table on the state of non-Gregorian support in Moment alternatives

**Comment by aliazizi** [comment]
Yes, I actually stuck in calculating a number of days in month in persian calendar.

**Comment by aliazizi** [comment]
After some search I found out there is no good alternative way if you want to convert calendar. The best library for this right now is moment.

For native support there is a proposal https://github.com/tc39/proposal-temporal but its not final yet.

There is a polyfill for that as well but still under development.

**Comment by bcullman** [comment]
@aliazizi - fee free to open a PR that adds a column to the existing readme.md table.
